index.bs

```diff
@@ -2802,6 +2802,26 @@ Note: Invoking this method from a [=browsing context=] where the [=Web Authentic
 
 </div>
 
+### Get Client Capabilities - PublicKeyCredential's `getClientCapabilities()` Method ### {#sctn-getClientCapabilities}
+
+<div link-for-hint="WebAuthentication/getClientCapabilities">
+
+[=[WRPS]=] use this method to determine the availability of a limited set of [=WebAuthn Client|client=] capabilities to offer certain workflows and experiences to users. For example, an RP may offer a sign in button on clients where only {{AuthenticatorTransport/hybrid}} transport is available (instead of showing a username field), or offer security key enrollment on [=WebAuthn Client|clients=] that support client PIN entry.
+
+Upon invocation, the [=client=] employs a [=client platform=]-specific procedure to discover availablity of these capabilities.
+
+This method has no arguments and returns a set of strings.
+
+<xmp class="idl">
+    partial interface PublicKeyCredential {
+        static Promise<ClientCapabilities> getClientCapabilities();
```

Comment:
Should this return a promise for a _sequence_ of `ClientCapabilities`? (Or, actually, of `DOMString` given [2.1.1](https://www.w3.org/TR/webauthn-2/#sct-domstring-backwards-compatibility).)